bundle/edu.gemini.spModel.io/src/test/scala/edu/gemini/spModel/io/impl/migration/to2016B/TargetMigrationTest.scala

```diff
@@ -102,7 +102,7 @@ class TargetMigrationTest extends Specification with MigrationTest {
     "Migrate MPC Minor Planets" in withTestProgram2("targetMigration.xml") { p =>
       p.findBaseByObsTitle("mpc-minor-planet") must_== Some(NonSiderealTarget(
         "beer",
-        IMap(1457994485000L -> Coordinates.fromDegrees(88.769, 21.08299999999997).get),
+        Ephemeris(Site.GN, IMap(1457994485000L -> Coordinates.fromDegrees(88.769, 21.08299999999997).get)),
```

Comment:
This is probably a stupid question, but is there any difference between using `IMap` and `==>>`?
I don't see one, but am curious why `IMap` would be essentially a type alias for `==>>`.